lib/vanagon/component/dsl.rb

```diff
@@ -270,6 +270,10 @@ def url(the_url)
         @component.url = the_url
       end
 
+      def upstream_url(the_url)
+        @component.options[:upstream_url] = the_url
```

Comment:
What's the benefit to using `@component.options[:upstream_url]` instead of `@component.upstream_url` since it's defined as an accessor?